dandiapi/api/mail.py

```diff
@@ -1,14 +1,21 @@
 import logging
-from typing import List
+from typing import Any, Dict, List
 
 from allauth.socialaccount.models import SocialAccount
+from django.conf import settings
 from django.contrib.auth.models import User
 from django.core import mail
-from django.template.loader import render_to_string
+from django.template.loader import render_to_string as _render_to_string
 
 logger = logging.getLogger(__name__)
 
 
+def render_to_string(template: str, render_context: Dict[str, Any]):
+    render_context['dandi_api_url'] = settings.DANDI_API_URL
+    render_context['dandi_gui_url'] = settings.DANDI_WEB_APP_URL
```

Comment:
What is this `render_context` object used for, and why does it have a different name for what I would expect to be `dandi_web_app_url`?

Reply:
`render_context` is a dictionary of variables passed to the Django templating engine that provides the variables the templates use. Anytime you see `{{ <variable_name> }}` in a template, it's getting that variable from the render context object.

Reply:
> What is this `render_context` object used for, and why does it have a different name for what I would expect to be `dandi_web_app_url`?

Yeah it should be `dandi_web_app_url`, I originally had the Django setting called `DANDI_GUI_URL` but I renamed it and forgot to update the context variables. It's fixed now.